LayoutTests/fast/mediastream/mediastreamtrack-configurationchange.html

```diff
@@ -29,6 +29,9 @@
         });
 
         assert_equals(track.label, "Mock audio device 2");
+
+        await new Promise(resolve => setTimeout(resolve, 2000));
+        assert_equals(track.readyState, "live");
```

Comment:
Rather than always waiting for two seconds, when most of the time that is much too long and sometimes it will be too short, it might be better to poll the attribute and fail if it doesn't change after some longer time.

Reply:
Here, we need to wait for the verification timer to kick in, so we need to wait 1 second.
I put 2 seconds just in case.
After 2 seconds, if the track is still live, we have recovered properly.